src/WrappedMToken.sol

```diff
@@ -520,7 +532,8 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
      * @param currentIndex_ The current index.
      */
     function _transfer(address sender_, address recipient_, uint240 amount_, uint128 currentIndex_) internal {
-        _revertIfInvalidRecipient(recipient_);
+        _revertIfZeroAccount(sender_);
+        _revertIfZeroAccount(recipient_);
```

Comment:
What if you want to burn your tokens?

Reply:
While I agree that it should be possible to purposely and your tokens to the zero address, every auditor send to care about preventing detrimental bag inputs, like a zero address for recipient, in transfers. Nevertheless, this logic isn't changing at all as the current M and wM already does not allow transferring to the zero address. This only adds the zero sender check for completeness.